rocket-chat-android-widgets/src/main/res/layout/message_inline_attachment.xml

```diff
@@ -93,14 +96,12 @@
 
         </LinearLayout>
 
-        <ImageView
+        <com.facebook.drawee.view.SimpleDraweeView
                 android:id="@+id/image"
-                android:layout_width="wrap_content"
-                android:layout_height="wrap_content"
-                android:maxHeight="200dp"
+                android:layout_width="match_parent"
+                android:layout_height="200dp"
```

Comment:
We could add some code to resize the view after we have the image. There's some example code of it done by a Fresco dev.

But they don't recommend that. Anyway it could be a temporary fix until the server gives us the image size.